cluster/stone-prd-rh01/tenants/rhtap-migration-tenant/hac-standard.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: hac-standard
+  namespace: rhtap-releng-tenant
+spec:
+  configuration:
+    include:
+      - '@redhat'
+    exclude:
+      - base_image_registries.base_image_permitted
+      - labels.disallowed_inherited_labels:description
+      - labels.disallowed_inherited_labels:io.k8s.description
+      - labels.disallowed_inherited_labels:io.k8s.display-name
+      - labels.disallowed_inherited_labels:io.openshift.tags
+      - labels.disallowed_inherited_labels:summary
+      - tasks.pinned_task_refs:create-frontend-dockerfile
+      - tasks.pinned_task_refs:parse-build-deploy-script
```

Comment:
What are these policies doing? Why are we excluding them?

Reply:
it appears inside the violations as the following:
```  - metadata:
      code: tasks.pinned_task_refs
      collections:
      - redhat
      depends_on:
      - tasks.pipeline_has_tasks
      description: Ensure that all Tasks in the SLSA Provenance attestation use an
        immuntable reference to the Task definition. To exclude this rule add "tasks.pinned_task_refs:create-frontend-dockerfile"
        to the `exclude` section of the policy configuration.
      solution: Make sure the build pipeline uses Tasks via pinned references. For
        example, if the git resolver is used, use a commit ID instead of a branch
        name.```
        it claims that our build pipelines does not have/use the tasks via pinned references

Reply:
I think that's one we can actually follow. We just need to use the git commit SHA instead of the `main` branch in the resolver. Let's remove it from the policy for now